cloudquery/sdk/transformers/openapi.py

```diff
@@ -24,11 +24,24 @@ def oapi_type_to_arrow_type(field) -> pa.DataType:
         return pa.string()
 
 
-def oapi_definition_to_columns(definition: Dict) -> List[Column]:
+def get_column_by_name(columns: List[Column], name: str) -> Column:
+    for column in columns:
+        if column.name == name:
+            return column
+    return None
+
+
+def oapi_definition_to_columns(definition: Dict, override_columns=[]) -> List[Column]:
     columns = []
     for key, value in definition["properties"].items():
         column_type = oapi_type_to_arrow_type(value)
-        columns.append(
-            Column(name=key, type=column_type, description=value.get("description"))
+        column = Column(
+            name=key, type=column_type, description=value.get("description")
         )
+        override_column = get_column_by_name(override_columns, key)
```

Comment:
Probably `override_columns.find(key)` or something like that should also work here